acceptance/tests/sameness/sameness_test.go

```diff
@@ -428,21 +428,30 @@ func TestFailover_Connect(t *testing.T) {
 				"../fixtures/cases/sameness/static-server/dc3")
 
 			// Create static client deployments.
+			staticClientDefaultKustomizeDir := "../fixtures/cases/sameness/static-client/default-partition"
+			staticClientAP1KustomizeDir := "../fixtures/cases/sameness/static-client/ap1-partition"
```

Comment:
~~ Total nit, but it might be easier to scan similar names like these if we put the different segment last, like
```suggestion
			staticClientKustomizeDirDefault := "../fixtures/cases/sameness/static-client/default-partition"
			staticClientKustomizeDirAP1 := "../fixtures/cases/sameness/static-client/ap1-partition"
```